src/fields/Assets.php

```diff
@@ -320,6 +320,14 @@ public function getElementValidationRules(): array
         return $rules;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function useFieldset(): bool
+    {
+        return true;
+    }
```

Comment:
This should probably happen in `BaseRelationalField` so Entries, Users, Categories, and Tags fields get updated as well.